internal/servers/controller/controller.go

```diff
@@ -90,6 +94,7 @@ func New(ctx context.Context, conf *Config) (*Controller, error) {
 		workerAuthCache:         new(sync.Map),
 		workerStatusUpdateTimes: new(sync.Map),
 		enabledPlugins:          conf.Server.EnabledPlugins,
+		apiListeners:            make([]*base.ServerListener, 0),
 	}
```

Comment:
Should we default `StartServiceUnavailableReplies` to something safe here, so that calling it without registering the health service doesn't panic?

Reply:
As part of the startup process (in `startListeners`), this handler is always set, but yeah let's set that to something just in case.